Electrino/Electrino/ENOJavaScriptApp.m

```diff
@@ -88,6 +126,18 @@ - (id)init
     return self;
 }
 
+// Create a new context for just evaluating the file
+// ISSUE: JSContext does not include -copyWithZone: method, so we have to manually copy the required methods.
+-(JSContext*)newContextForEvaluation
+{
+	JSContext* newContext = [[JSContext alloc] initWithVirtualMachine:self.jsVM];
+	newContext[@"require"] = self.jsContext[@"require"];
+	newContext[@"process"] = self.jsContext[@"process"];
+	newContext[@"console"] = self.jsContext[@"console"];
+	[newContext evaluateScript:@"var exports = {};"]; // Evaluated so the developer doesnt have to
```

Comment:
First, great work. This is a big step forwards.

To mimic Node.js behavior you should inject a `module` object that has an `exports` object initialized as an empty object. Pseudo-code:

```objc
module = @{ @"exports": @{} }
newContext[@"module"] = module;
newContext[@"module"] = module.exports;
// 1.- evaluate
// 2.- pick the evaluated exports
return (id)[tmpContext objectForKeyedSubscript:@"module.exports"];
```

This way these two ways of populating `exports` will work:
```
exports.something = foo
module.exports = somethingCompletelyNew
```